corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-select-form-control.md

```diff
@@ -0,0 +1,6 @@
+Please replace form-control with form-select.
+
+In Bootstrap 3, selects used the `form-control` class.
+In Bootstrap5, selects use `form-select` instead.
+
+Please remove `form-control` from selects and add `form-select`.
```

Comment:
Is this line convey the same meaning as first line?

Reply:
good point - 9966da6eeb7f28c54608766ac15df69ed91d1d2c